custom-vu/src/main/kotlin/jces1209/vu/TrafficDataParser.kt

```diff
@@ -0,0 +1,22 @@
+package jces1209.vu
+
+import java.io.StringReader
+import javax.json.Json
+import javax.json.JsonReader
+
+class TrafficDataParser {
+
+    companion object {
+
+        fun parseData(hostName: String, readEnvTrafficShapeConfig: String): String {
+
+            val jsonReader: JsonReader = Json.createReader(StringReader(readEnvTrafficShapeConfig))
+            val obj: javax.json.JsonObject = jsonReader.readObject()
```

Comment:
it's best to use `use` method here:
```suggestion
           val propertiesFileName
           jsonReader.use {
               val obj: javax.json.JsonObject = jsonReader.readObject()
               propertiesFileName: String = obj.getString(hostName)
           }
           return propertiesFileName
```
Since `JsonReader` implements `AutoCloseable` and `Closeable`.

Reply:
Thanks for suggesting this Alex, done!